### DISCUSSION TITLE
How to stock skill 'shop' in events

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#348] [General]
I'm trying to create a skill shop using eventing, but the entire skill list just defaults to hp+5 (the very first skill ID). All skills chosen are within the 0xFF range, so I'm curious what the issue is here 🤔 

![image](https://github.com/user-attachments/assets/ee9f8964-d287-4443-9c16-9a6a08238e5d)

![image](https://github.com/user-attachments/assets/195c01bd-6551-4641-82f5-fd80f2b1daf2)

**Comment by MokhaLeee** [comment]
it is a issue on vanilla code. shop will neglect the item uses

**Reply by JesterWizard** [reply]
I'm not sure how this answers my question? It's not the item uses I'm concerned about, it's the fact the skills I've chosen to add to the store are just completely ignored 🤔

**Reply by MokhaLeee** [reply]
Since shop related code is not touched, so you need to dig on vanilla code to find the answer.

**Reply by MokhaLeee** [reply]
Is it worked fine in old system?

**Reply by JesterWizard** [reply]
It does yeah. PokeEmblem most notably has a working shop system for skills. Maybe I can use that as reference.
